### DISCUSSION TITLE
Portal Gun styles.

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4125] [Feature Suggestions]
Like chamber styles. But you can choose a portal gun skin, like default co-op and classic or P1 styled. Also this affects the long fall boots if supported.

**Comment by slambo311** [comment]
This would be pretty cool. Maybe even a cleaner portal gun (Very original idea, I know. Definitely not 81932 clean portal gun reskins on GameBanana) could be put in. Of course, custom portal gun mods do exist, so a "No override" option might be needed.

**Reply by SimpleFoxOfficial** [reply]
Reskins cool too, OR make the custom palletes, where you can add your own portal gun texture and save it in pallet.

**Reply by slambo311** [reply]
That would be pretty cool. Maybe also adding your own portal gun model, too.

**Comment by vrad-exe** [comment]
We have the P1 gun in P1 style, we don't plan to add any other portal gun overrides right now, except maybe the potato gun as part of certain voice packs

**Reply by SimpleFoxOfficial** [reply]
what about the PS:M portal gun and Reloaded portal gun?

**Reply by slambo311** [reply]
> what about the PS:M portal gun and Reloaded portal gun?

One word: Copyright.

**Comment by CN1o2** [comment]
That's a very good idea. Maybe it's not that hard to add them.

**Reply by FrozenDragon0** [reply]
There is also a limit on how much custom content can be loaded at once IIRC.

**Reply by slambo311** [reply]
I don't know, just coding in general is pretty hard.

**Reply by vrad-exe** [reply]
There is very little actual programming involved in this, please stop acting like you understand the engine when you clearly don't.

**Reply by slambo311** [reply]
I do understand a little, but alright.
